cinema/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(max_length=255)
```

Comment:
Consider using `serializers.CharField()` without specifying `max_length=255` for the `description` field if you plan to change the model field to `TextField` as suggested in the `models.py` review.